src/frontend/components/timer.tsx

```diff
@@ -33,9 +33,6 @@ export default function Timer({
       setTimeLeft(time);
     }
   }
-  return (
-    <div className={`${styles.timer} ${customClass}`}>
-      {formatTime(timeLeft)}
-    </div>
-  );
+
+  return <div className={styles.timer}>{formatTime(timeLeft)}</div>;
```

Comment:
My only comment is that the only place this actual timer component is used is in the waiting room, do we want that count down to flash red when it gets low? There's not really anything the user can do about that

Reply:
Yea it's only used in the waiting room so I already removed the countdown red color change from the timer component.

Reply:
Great, then if it looks good to you then i say go for it